lib/job-simulation.rb

```diff
@@ -6,9 +6,37 @@ class JobSimulation
 
   def initialize (jobs_available, job_seekers)
 
+      @workers = Stack.new
+      @waiting = Queue.new
+
+      job_seekers.times do |worker|
+        #establish initial Employed list (Stack)
+        if worker < jobs_available
+          @workers.push("Worker #{worker + 1}")
+        else
+          #establish initial Waitlist (Queue)
+          @waiting.enqueue("Worker #{worker + 1}")
+        end
+      end
```

Comment:
Nice solution! I haven't seen someone else do that in C7 yet! Very clean and clear.